dev-haskell/hw-kafka-client/hw-kafka-client-5.0.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.2.0
```

Comment:
Adding a `#hackport` line like this will keep the `it` flag disabled on later updates:

```suggestion
# ebuild generated by hackport 0.8.2.0
#hackport: flags: -it
```

or if you wanted to rename the flag to `test`:

```suggestion
# ebuild generated by hackport 0.8.2.0
#hackport: flags: it:test
```